tests/backends/tests.py

```diff
@@ -745,6 +745,19 @@ def test_cursor_contextmanager_closing(self):
             self.assertIsInstance(cursor, CursorWrapper)
         self.assertTrue(cursor.closed)
 
+    @skipUnlessDBFeature('test_db_allows_multiple_connections')
```

Comment:
How does `test_db_allows_multiple_connections` relate to `is_usable()`? It seems like you'd want to be including the test in each of the backends that you added the code to.

Alternatively or in addition, it seems like you can add a test unconditionally for _all_ backends that calling `is_usable()` doesn't raise an exception.